### PR TITLE
chore(dynamic-plugins-info): update readme

### PR DESCRIPTION
**Pull request by gashcrumb**
This change updates the readme for the dynamic-plugins-info plugin to include installing the plugin via helm and adds a screenshot.  Part of [RHIDP-167](https://issues.redhat.com/browse/RHIDP-167)

### CODE REVIEW

File: plugins/dynamic-plugins-info/images/screenshot.png
Comment (invincibleJai) [inline plugins/dynamic-plugins-info/images/screenshot.png:1]: good to add some name to the screenshot i.e pluginList.png or something similar

Comment (gashcrumb) [reply plugins/dynamic-plugins-info/images/screenshot.png:1]: Yup good call, updated that.

**Pull request merged**